### DISCUSSION TITLE
File Dropzone

### DISCUSSION THREAD

**Discussion by MacelinCane** [mdbootstrap/TW-Elements#2328] [Share ideas & request features]
It's always a big mess when trying to design and implement a decent file dropzone, for media especially, with thumbnails (with listing or tiling for multiple) and showing or not filenames, sizes and delete options, etc..  It would be great to have min and max limits, and probably some other things that a backend person could add. (that's not me 😊 )
I think your team could do a great job with this!

![image](https://github.com/mdbootstrap/TW-Elements/assets/57817530/66b44fda-e4ec-4902-9fd1-ee1df6552094)

**Comment by iprzybysz** [comment]
Hi @MacelinCane, thanks for your proposal. I've added it to our ideas list.
